docs/source/reporters.rst

```diff
@@ -210,6 +211,30 @@ Embedded content might be easier to read and identify individual reports. Subjec
 
 When `colored` is true reports will be embedded in code section (with diff syntax) to enable colors.
 
+Gotify
+------
+
```

Comment:
Maybe link to https://gotify.net and/or at least say what Gotify is?

Reply:
Short description with a link added.